### PR TITLE
kdevelop: trigger build

### PR DESCRIPTION
**Pull request by kenmays**
Verified build on hrev54415 x86_64.
Closes: https://github.com/haikuports/haikuports/issues/5190

### CODE REVIEW

File: dev-util/kdevelop/kdevelop-5.4.6.recipe [hunk 1]
```diff
@@ -7,7 +7,7 @@ HOMEPAGE="https://www.kdevelop.org/"
 COPYRIGHT="2010-2019 KDE Organisation"
 LICENSE="GNU GPL v3
 	GNU LGPL v3"
-REVISION="2"
+REVISION="1"
```
Comment (Begasus) [inline dev-util/kdevelop/kdevelop-5.4.6.recipe:10]: Shouldn't this be revision 3?

Comment (extrowerk) [reply dev-util/kdevelop/kdevelop-5.4.6.recipe:10]: Awww...

**Comment by Kitsune64** [comment]
I don't understand why you closed my Issue. That was on x86 system (32bits) about a missing library when installing by Haiku Depôt. It don't work for now...There is a way to make it work?

**Comment by kenmays** [comment]
@Kitsune64 0 Kdevelop was bumped to 5.4.6, so that is the current supported build., We are phasing out porting support for the older LLVM 5.x build.

**Comment by Kitsune64** [comment]
> Kdevelop was bumped to 5.4.6, so that is the current supported build., We are phasing out porting support for the older LLVM 5.x build

Okay so here I have here the version available 5.4.1-1 show in the list of Haiku Depôt after a Haiku update: R1/beta2 hrev54154+116 x86_gcc2.

**Pull request merged**